src/main/kotlin/carracing/Race.kt

```diff
@@ -1,16 +1,17 @@
 package carracing
 
+import kotlin.random.Random
+
 class Race(
-    private val numCars: Int,
+    carNames: List<String>,
     private val numRounds: Int,
 ) {
-    private val cars = List(numCars) { Car() }
+    private val cars = carNames.map { Car(it) }
 
     fun start() {
         for (round in 1..numRounds) {
-            cars.forEach { it.move() }
+            cars.forEach { it.move(Random.nextInt(10)) }
```

Comment:
What do you think are the advantages of using an interface when working with objects that are hard to test, like those involving randomness?

Reply:
Do you think using an interface in this situation could provide any particular benefits?